docs/UserGuide.md

```diff
@@ -194,6 +197,20 @@ Format: `filter t/KEYWORD t/[MORE_KEYWORDS]...`
 
 Example: `filter t/friends t/family` Lists all persons in the address book whose tags match any of the specified keywords.
 
+### Adding a wedding : `addw`
+Adds a wedding to the address book.
+
+Format: `addw n/WEDDING NAME c/CLIENT [d/DATE] [v/VENUE]`
+
+Tip: CLIENT can be inputted by index or name.
+
+Examples:
+* `addw n/Alice's Wedding c/Alice`
+* `addw n/Alice's Wedding c/1`
+* `addw n/Bob's Wedding c/Bob d/2024-12-31`
+* `addw n/Bob's Wedding c/Bob v/Sentosa`
+* `addw n/Bob's Wedding c/Bob d/2024-12-31 v/Sentosa`
```

Comment:
Maybe can add one where
`addw n/Bob's Wedding c/Bob d/2024-12-31 v/Sentosa`
requires
another command after due to duplicates of bob
`addw n/Bob's Wedding c/2 d/2024-12-31 v/Sentosa`